how-to-be-a-developer/readme.md

```diff
@@ -0,0 +1,26 @@
+> Goal of this guide: learn about what the role of a developer is at New Atoms
+
+## It's all about the perspective
+
+Being a developer is a way of looking at certain situations.
+For example: if a client wants to add something to a produced article, the conclusions drawn from this request can be very different depending on your perspective as a certain developer:
+* Was there anything wrong with the content? [content]
+* How, if applicable, can the company structurally integrate this feedback into the production process? [operations]
+* What effect will ignoring the request have on the relationship with the costumer? [business]
+* How can the experience of giving feedback be optimized? [experience]
```

Comment:
> [experience]

there should be developer behind it because you only later on explain the types of developer. Now there's is too little context to understand what's between the brackets.